Sources/Sentry/SentrySDK.m

```diff
@@ -19,6 +20,7 @@ @implementation SentrySDK
 
 static SentryHub *currentHub;
 static BOOL crashedLastRunCalled;
+static SentryAppStartMeasurement *appStartMeasurement;
```

Comment:
I believe access to this field must be synchronized. Since we're setting it once and not modifying it anymore, I believe a double-check lock might be a good approach

Also, there's a property declared on the private header but we have actual an actual getter and setter here so I'm a bit confused.

Reply:
The property in the private header is a way to make the getter and setter in SentrySDK.m visible. The private header is actually a category. You can't add properties in categories. Therefore we need an actual implementation in the SentrySDK.m.